src/components/selfServicePage/SelfServicePage.js

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import Grid from '@material-ui/core/Grid';
+import Paper from '@material-ui/core/Paper';
+import Button from '@material-ui/core/Button';
+import {useStyles} from '../LayoutStyles';
+import mockAuth from '../Authenticator';
+import MemberSelfService from './MemberSelfService';
+import {Link} from 'react-router-dom';
+import InviterSelfService from './InviterSelfService';
+
+export default function SelfServicePage() {
+  const classes = useStyles();
+
+  return (
+      <Paper className={classes.paper}>
+        <Grid
+            container
+            direction="column"
+            justify="flex-start"
+            alignItems="stretch"
+            className={classes.gridContainer}>
+          {mockAuth.isInviter() ? <InviterSelfService/> : <MemberSelfService/>}
```

Comment:
I thought an inviter could be also a member, no?

Reply:
Inviter == Googler in our case.
They don't have a reason to add themselves to mailing lists, their role is to accept their spouses. Am I right?

Reply:
> their role is to accept their spouses

yes, in our app and in real life :joy:

That would mean, two partnered inviters could not be members at the same time. Not sure if they would like to be.

Reply:
Actually they can be. They will just log in with their @gmail accounts, not @google accounts. It makes perfect sense since it's an external group and using work account shouldn't be necessary for personal usage. I think we were talking about it during one of the meetings and the general conclusion was (if I remembered it correctly) that the worst thing that can happen is Googler inviting themself and we decided to ignore it.

Reply:
I accidentally tagged someone 😁

Reply:
hehe, ok true I forgot this workflow with different emails.